### PR TITLE
fmod stuff

### PR DESCRIPTION
**Pull request by TheSillyDoggo**

### CODE REVIEW

File: bindings/2.204/GeometryDash.bro [hunk 1]
```diff
@@ -3636,8 +3636,14 @@ class FMODAudioEngine : cocos2d::CCNode {
 	TodoReturn countActiveEffects();
 	TodoReturn countActiveMusic();
 	TodoReturn createStream(gd::string) = win 0x3aea0;
-	TodoReturn disableMetering();
-	TodoReturn enableMetering();
+	TodoReturn disableMetering()
+	{
+		*(reinterpret_cast<bool*>(reinterpret_cast<uintptr_t>(this) + 0x188)) = false;
```
Comment (SpaghettDev) [inline bindings/2.204/GeometryDash.bro:3641]: MBO jumpscare

**Comment by SpaghettDev** [comment]
use the members i sent you xd

**Comment by TheSillyDoggo** [comment]
> use the members i sent you xd

They didn't work xd

**Review by matcool — CHANGES_REQUESTED** [review]
dont use hardcoded offsets for members, follow our style guides (we put braces on the correct line)

**Pull request closed without merging**